src/widgets/breadcrumbs/ui/breadcrumb.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames/bind';
+import { Link } from 'react-router-dom';
+
+import styles from './breadcrumbs.module.scss';
+
+const cx = classNames.bind(styles);
+
+interface BreadcrumbProps {
+  linkTo: string;
+  text: string;
+  isLastLink?: boolean;
+}
+
+export function Breadcrumb(props: BreadcrumbProps) {
+  const { linkTo, text, isLastLink } = props;
```

Comment:
```suggestion
export function Breadcrumb({ linkTo, text, isLastLink }: BreadcrumbProps) {
```
Maybe we can do destructuring here? similar to other components

Reply:
fixed in https://github.com/rolling-scopes/site/pull/516/commits/e49804e51e5eb37e7d12fe448dbabe83738e5535